src/app/settings/edit-email/page.tsx

```diff
@@ -0,0 +1,58 @@
+'use client';
+import React, { useState } from 'react';
+import styles from './page.module.css';
+import InputField from '@/components/textbox/textbox';
+import ActiveButton from '@/components/Active-Button/ActiveButton';
+import Image from 'next/image';
+import backIcon from '@/assets/BackArrow.svg';
+
+import Link from 'next/link';
+
+const Editaccount: React.FC = () => {
+  const [email, setEmail] = useState('');
+  const [textboxentry, settextboxentry] = useState(false);
+
+  const handleEmailChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const value = e.target.value;
+    setEmail(value);
+    if (value) {
+      settextboxentry(true);
+    } else {
+      settextboxentry(false);
+    }
+  };
+  return (
+    <div className={styles.editaccountconatiner}>
+      <div className={styles.backButtonContainer}>
+        <Link href="/settings">
+          <Image
+            priority
+            src={backIcon}
+            alt="Back"
+            className={styles.backButtonImage}
+          />
+        </Link>
+      </div>
+      <h1 className={styles.editAccounth1}>Edit Account</h1>
+      <div className={styles.inputfieldContainer}>
+        <InputField
+          type="email"
+          value={email}
+          onChange={handleEmailChange}
+          placeholder="Enter email"
+          label="Password"
+        />
+      </div>
```

Comment:
avoid having divs across your cold that only have one element inside going forward. this introduces prop drilling and not only is this a problem with maintaining the code, but it also is more likely to introduce bugs in your CSS

Delete the div tag

Make sure to do this for any other elements on this page that have a div with only one element.